modules/dists/BlockCycDist.chpl

```diff
@@ -444,10 +446,8 @@ proc BlockCyclic.init(other: BlockCyclic, privatizeData,
 
   this.complete();
 
-  for i in targetLocDom {
-    targetLocales[i] = other.targetLocales[i];
-    locDist[i] = other.locDist[i];
-  }
+  targetLocales = other.targetLocales;
```

Comment:
I think you and @vasslitvinov should be aware of each other's PRs that change initialization of these kinds of fields.

Reply:
On a quick look at https://github.com/chapel-lang/chapel/pull/15044, it looks like there won't be any logic conflicts, but there might be merge conflicts. I'll touch base with him. Thanks for pointing it out!